packages/core/contracts/optimistic-asserter/implementation/OptimisticAsserter.sol

```diff
@@ -238,6 +232,13 @@ contract OptimisticAsserter is OptimisticAsserterInterface, Lockable, Ownable {
         }
     }
 
+    function syncUmaParams(bytes32 identifier, address currency) public {
+        cachedUmaParams.oracle = finder.getImplementationAddress(OracleInterfaces.Oracle);
+        cachedUmaParams.identifiers[identifier] = _getIdentifierWhitelist().isIdentifierSupported(identifier);
+        cachedUmaParams.currencies[currency].isWhitelisted = _getCollateralWhitelist().isOnWhitelist(currency);
+        cachedUmaParams.currencies[currency].finalFee = _getStore().computeFinalFee(currency).rawValue;
+    }
```

Comment:
Should we emit an event?

Reply:
I was not adding events as this is more of an internal implementation on how OA gets UMA system settings in a gas optimized way. Also this can be called with zeroed parameters and we would need to implement additional logic to separate them in 3 separate events.